service/discovery/discovery.go

```diff
@@ -296,15 +296,16 @@ func (svc *Service) Start(ctx context.Context, req *discovery.StartDiscoveryRequ
 	}
 
 	for _, v := range discoverer {
-		log.Infof("Scheduling {%s} to execute every {%d} minutes...", v.Name(), svc.discoveryInterval)
+		log.Infof("Scheduling {%s} to execute every {%f} minutes...", v.Name(), svc.discoveryInterval.Minutes())
```

Comment:
I think you can just use `%s` directly with the interval, this should pretty print it in something like `5m` or `10h5m`, depending on the interval